src/components/graph-view.js

```diff
@@ -1581,11 +1584,18 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
       this.entities.appendChild(nodeContainer);
     }
 
-    // ReactDOM.render replaces the insides of an element This renders the element
-    // into the nodeContainer
     const anyElement: any = element;
 
-    ReactDOM.render(anyElement, nodeContainer);
+    let root = this.roots[containerId];
+
+    if (!this.roots[containerId]) {
```

Comment:
nit: This could just be `if (!roots)` and similar below. Otherwise LGTM

Reply:
oh i forget to update this logic after updating line 1589. Will update.